config/initializers/assets.rb

```diff
@@ -26,6 +26,4 @@
 Rails.application.config.assets.precompile += %w( potlatch2/FontLibrary.swf )
 Rails.application.config.assets.precompile += %w( potlatch2/locales/*.swf )
 Rails.application.config.assets.precompile += %w( help/introduction.* )
-Rails.application.config.assets.precompile += %w( iD/img/*.svg iD/img/*.png iD/img/*.gif )
-Rails.application.config.assets.precompile += %w( iD/img/pattern/*.png )
-Rails.application.config.assets.precompile += %w( iD/locales/*.json )
```

Comment:
Well maybe, but there's all sorts of stuff in there, much of which I wonder whether we even need at all, never mind being able to serve it as assets...